appinventor/components/src/com/google/appinventor/components/runtime/ContextMenu.java

```diff
@@ -0,0 +1,107 @@
+// -*- mode: java; c-basic-offset: 2; -*-
+// Copyright 2020 MIT, All rights reserved
+// Released under the Apache License, Version 2.0
+// http://www.apache.org/licenses/LICENSE-2.0
+
+package com.google.appinventor.components.runtime;
+
+import android.app.Activity;
+import android.view.View;
+import com.google.appinventor.components.annotations.DesignerComponent;
+import com.google.appinventor.components.annotations.SimpleObject;
+import com.google.appinventor.components.common.ComponentCategory;
+import com.google.appinventor.components.common.YaVersion;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Container for ContextMenuItems
+ * @author singhalsara48@gmail.com (Sara Singhal)
+ *
+ */
+@DesignerComponent(version = YaVersion.CONTEXT_MENU_COMPONENT_VERSION,
+        category = ComponentCategory.LAYOUT,
+        description = "Component for context menu to hold ContextMenuItems.")
+@SimpleObject
+public class ContextMenu implements Component, ComponentContainer, View.OnCreateContextMenuListener {
+
+    private Form form;
+    private android.view.ContextMenu menu;
+
+    private List<MenuItem> items;
+
+    /**
+     * Create a new Context Menu component.
+     *
+     * @param form The form that will render this menu.
+     */
+    public ContextMenu(Form form) {
+        items = new ArrayList<>();
+        form.registerForOnCreateContextMenu(this);
+        this.form = form;
+    }
+
+    @Override
+    public void onCreateContextMenu(android.view.ContextMenu menu, View view, android.view.ContextMenu.ContextMenuInfo menuInfo) {
```

Comment:
You can import `android.view.ContextMenu.ContextMenuInfo` so you don't have to fully qualify it here.